meet-data/spf-archive/0414/lifters.csv

```diff
@@ -0,0 +1,48 @@
+Name,Age,Division,WeightClassKg,BestSquatKg,BestBenchKg,BestDeadliftKg,TotalKg,Sex,Event,Equipment,Place
+Brianna Yoho,16,Teenage (16-17),52,77.11,47.63,88.45,213.19,F,SBD,Wraps,1
+Alyssa Hawkins,29,Police and Fire,60,90.72,74.84,124.74,290.3,F,SBD,Wraps,1
+Franky Moore,28,Police and Fire,82.5,106.59,70.31,158.76,335.66,F,SBD,Wraps,1
+Kathryn Kahdy,32,Open,48,97.52,58.97,102.06,258.55,F,SBD,Wraps,2
+Rachel Bess,24,Open,48,113.4,74.84,136.08,324.32,F,SBD,Wraps,1
+Madelaine Fellela,21,Open,52,97.52,54.43,99.79,251.74,F,SBD,Wraps,1
+Brittany Satterwhite,25,Open,56,113.4,58.97,120.2,292.57,F,SBD,Wraps,1
+Adee Zukier,24,Open,67.5,124.74,68.04,147.42,340.19,F,SBD,Wraps,1
+Erin Young,24,Open,67.5,127.01,79.38,129.27,335.66,F,SBD,Wraps,2
+Tara Highbaugh,34,Open,67.5,81.65,72.57,106.59,260.82,F,SBD,Wraps,3
+Tara Highbaugh,34,Open,67.5,,81.65,,81.65,F,B,Raw,1
```

Comment:
This line is weird... I'm pretty sure SPF just made an error here, because her Bench-only number is the same as her Squat. I'll just take this line out.